lib/Rest/VstResponse.cpp

```diff
@@ -163,6 +163,12 @@ ErrorCode VstResponse::deflate() {
   return TRI_ERROR_INTERNAL;
 }
 
+ErrorCode VstResponse::gzip() {
+  // we should never get here
+  TRI_ASSERT(false);
```

Comment:
tests/js/client/shell/api/multi/foxx-spec.js will probably demand this.

Reply:
I ran it and it didn't fail.